packages/migration/migrations/hearth/20230220081813-update-indices.js

```diff
@@ -0,0 +1,133 @@
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: Encounter')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: Observation')
+      await db.collection('Observation').createIndex({ id: 1 })
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: Observation')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: DocumentReference')
+      await db.collection('DocumentReference').createIndex({ id: 1 })
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: DocumentReference')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: Practitioner')
+      await db.collection('Practitioner').createIndex({ id: 1 }) // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: Practitioner')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: PractitionerRole')
+      await db.collection('PractitionerRole').createIndex({ id: 1 })
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: PractitionerRole')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: RelatedPerson')
+      await db.collection('RelatedPerson').createIndex({ id: 1 })
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: RelatedPerson')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: Patient')
+      await db.collection('Patient').createIndex({ id: 1 })
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: Patient')
+
+      // eslint-disable-next-line no-console
+      console.log('Updating indices for collection: PaymentReconciliation')
+      await db.collection('PaymentReconciliation').createIndex({ id: 1 })
+      // eslint-disable-next-line no-console
+      console.log('Done updating indices for collection: PaymentReconciliation')
+    })
+  } finally {
+    await session.endSession()
+  }
+}
+
+export const down = async (db, client) => {
+  const session = client.startSession()
+  try {
+    await session.withTransaction(async () => {
+      await db.collection('Task').dropIndexes({ 'focus.reference': -1, id: -1 })
+      await db.collection('Composition').dropIndex({ id: -1 })
+      await db.collection('Location').dropIndex({ id: -1 })
+      await db.collection('Encounter').dropIndex({ id: -1 })
+      await db.collection('Observation').dropIndex({ id: -1 })
+      await db.collection('Composition').dropIndex({ id: -1 })
+      await db.collection('Encounter').dropIndex({ id: -1 })
+      await db.collection('Observation').dropIndex({ id: -1 })
```

Comment:
Encounter and Observation seems to be duplicated. Same in the up function as well

Reply:
Sorry for the typo, fixed in https://github.com/opencrvs/opencrvs-core/pull/4706/commits/270763159f7c733bf6296823867ad7560ffc0b0e